### PR TITLE
Update Tooltip Hover Feature

### LINKED ISSUE #121: Tooltip Hover Update
## Description

The hover for the tooltips on the Company Details page doesn't persist even if your cursor scans the space where the tooltip content would be. 

i.e. If i wanted to click on the capital goods link, the tooltip vanishes as soon as I try to move my cursor to click on that link.
image

## Todo

These points are a rough guideline. Please feel free to discuss with others on the team about the best way to design this component!

- [ ] Have tooltip hover persist over cursor scan of tooltip container

### PR DESCRIPTION
**Pull request by eeshabarua**
## Summary

Making a few lil bug fixes to update the Company Details Page.
Closes #121 

## Changes

- Tooltips now persist on hover of the Tooltip container
- Tooltip group flex wrap has spacing between first and second row
- NA -> N/A

## Screenshots

image

image

### CODE REVIEW

File: src/components/Tooltips/ToolTip.tsx [hunk 1]
```diff
@@ -14,42 +14,56 @@ const ToolTip: FC<TitleProps> = ({ title, details, children }) => {
...
-    <>
-      <div className="group relative z-40 flex w-fit justify-center">
-        <button
-          className={clsx('bg-primary rounded', 'text-base text-black')}
+    <div className="group relative flex w-fit justify-center">
+      <button
+        className={clsx('bg-primary rounded', 'z-10 text-base text-black ')}
+        onMouseEnter={() => setIsShown(true)}
+        onMouseLeave={(e) => {
+          if (
```
Comment (daniel-moon32) [inline src/components/Tooltips/ToolTip.tsx:22]: You should extract this into a separate function since you are reusing it below.

**Review by daniel-moon32 — CHANGES_REQUESTED** [review]
Looks good overall! Few things related to making sure the code is not repeated and more stylistically clean.

**Review by zorazrr — APPROVED** [review]

File: src/components/Tooltips/ToolTip.tsx [hunk 2]
```diff
@@ -13,43 +13,58 @@ interface TitleProps {
...
+  }
+
   return (
-    <>
-      <div className="group relative z-40 flex w-fit justify-center">
-        <button
-          className={clsx('bg-primary rounded', 'text-base text-black')}
+    <div className="group relative flex w-fit justify-center">
+      <button
+        className={clsx('bg-primary rounded', 'z-10 text-base text-black ')}
```
Comment (daniel-moon32) [inline src/components/Tooltips/ToolTip.tsx:34]: Nit, these shouldn't be separated by a comma if they aren't conditionally being changed.

File: src/pages/company/[id].tsx [hunk 3]
```diff
@@ -77,8 +77,8 @@ const Company = () => {
       <div className="mt-6 ml-8">
         <BackButton />
       </div>
-      <div className="mb-20 flex flex-col px-12 lg:px-24">
-        <div className="flex flex-col items-center">
+      <div className="mb-20 flex flex-col flex-wrap px-12 lg:px-24">
+        <div className="flex flex-col items-center ">
```
Comment (daniel-moon32) [inline src/pages/company/[id].tsx:81]: Remove space at the end of this string

File: src/components/Tooltips/ToolTip.tsx [hunk 4]
```diff
@@ -13,43 +13,58 @@ interface TitleProps {
...
-          </div>
-        )}
-      </div>
-    </>
+          />
+          {children ? (
+            children
+          ) : (
+            <>
+              <div className=" font-semibold">{title}:</div>
```
Comment (daniel-moon32) [inline src/components/Tooltips/ToolTip.tsx:61]: Hmm I think this is leftover but there shouldn't be a random space before the font-semibold

**Pull request merged**